R/get_gambl_metadata.R

```diff
@@ -47,6 +47,16 @@ get_gambl_metadata = function(seq_type_filter = "genome",
                               only_available = TRUE,
                               seq_type_priority = "genome"){
 
+  # check arguments
+  stopifnot('`tissue_status_filter` argument should be either "tumour" or "normal".' = {
+    identical(tissue_status_filter, "tumour") | identical(tissue_status_filter, "normal")
+  })
+  
+  stopifnot('`tissue_status_filter` argument should be either "tumour" or "normal".' = {
```

Comment:
This probably should have another message printed when the check is not passed? Based on the `seq_type_priority`

Reply:
Yes. It's fixed now. Thank you.